flang-rt/lib/runtime/descriptor.cpp

```diff
@@ -85,7 +85,7 @@ RT_API_ATTRS void Descriptor::Establish(int characterKind,
 RT_API_ATTRS void Descriptor::Establish(const typeInfo::DerivedType &dt,
     void *p, int rank, const SubscriptValue *extent,
     ISO::CFI_attribute_t attribute) {
-  std::size_t elementBytes{dt.sizeInBytes()};
+  std::size_t elementBytes{static_cast<std::size_t>(dt.sizeInBytes())};
```

Comment:
As a stylistic point, I usually use `auto` for declarations initialized with casts, since the type is obvious and doesn't need to appear twice.